### DISCUSSION TITLE
Ghostty's ghost mode

### DISCUSSION THREAD

**Discussion by AnirudhG07** [ghostty-org/ghostty#3032] [Feature Requests, Ideas] [answered]
This is just an idea, but having a cool ghost mode of "ghostty" would be SOO COOL!
Just throwing out ideas, what if the ASCII ghost animation on the ghostty website can be displayed on ghostty as `ghostty +ghost` or something.
What if you can have a screensaver with that or something ghost like.
What if u can have a dim mode with a circular blur radius  of light which makes the terminal text visible more and rest is very dim, you know those ghosty and night feeling.

Can anything super cool like this be supported somehow? Like a ghost mode.

**Comment by pluiedev** [comment, marked as answer]
There's a PR for it: #4876

**Reply by AnirudhG07** [reply]
Noice!
